components/Footer.tsx

```diff
@@ -65,6 +67,21 @@ const Footer = () => {
           {linkInfo.text}
         </a>
       </em>
+      <NextLink href="/privacypolicy" as="/privacypolicy">
+        <a style={{ borderBottom: 'none' }}>
+          <p role="term">
+            {' '}
+            {language === ENGLISH ? 'Privacy Policy' : 'política de privacidad'}
+          </p>
+        </a>
+      </NextLink>
+      <div style={{ display: 'flex', width: '95%', marginTop: '1.5rem' }}>
+        <em>
+          {language === ENGLISH
+            ? `Thrive is a free, nonprofit resource directory developed by Code for America and Santa Barbara County Public Defender's office for people who have been system impacted in Santa Barbara County. We are not a law firm and the information on this site is not legal advice.`
```

Comment:
@vsauceda-cfa I wonder if this might be better located on the privacy policy page itself instead of the footer? What do you think?

Reply:
I was following how clearmyrecord was doing it on their site. Im open to either way. Which do you think is better?

Reply:
@vsauceda-cfa

Looking again, I do think having a disclaimer at the bottom of every pages is a good idea. I think you may be over complicating the CSS a bit though. 🤔

Try taking the `em` tag out of the `div` tag you created, and make it a direct child of the `footer` element. Then open up `Footer.module.css` and delete the `height` rule on line 3. That should make the text nice and responsive without any chance of extra margins or gaps in spacing.

Lemme know how it goes! 🖖

Reply:
Ok fixed this. It should be good to go. And yeah with the footer test it looks like it is reading the last item and it keeps returning privacy policy so I just gave it what it was asking for. Hope its ok but if you need it to go back to the old test I could try to figure it out. Let me know. Enjoy. your evening